### ISSUE TITLE
architecture_026 fixable does not work

### ISSUE THREAD

**Issue by patrick-studer** [jeremiah-c-leary/vhdl-style-guide#1206]
**Environment**
v3.25

**Describe the bug**
architecture_026 is configured to be "fixable" but fix is not applied.
I keep getting the error message: "Move : 2 columns" for the 3 lines with less indentation

```
architecture rtl of test is

    signal WrPl_Data   : std_logic_vector(Wr_Data'range);
    signal WrPl_Vld    : std_logic;
    signal AxiWrCmd_Rdy  : std_logic;
    signal AxiWrDat_Rdy  : std_logic;
    signal AxiWrDat_Data : std_logic_vector(AxiDataWidth_g - 1 downto 0);
    signal WrPl_Rdy    : std_logic;

begin

end architecture;
```

**To Reproduce**
1. Take above snipped
2. Activate rule architecture_026 (fixable: true)
3. Run vsg --fix
4. See error (not automatically fixed)

**Comment by patrick-studer** [comment]
I guess I found my issue.

The fix is not applied if you have configured severity = Warning.

I don't know if this is wanted. I can run a report and get warnings of problems and then decide to fix them. But it looks like only errors may be fixed automatically.
I guess this is also somewhere documented :)

Therefore, I close again my wrong bug report.
